FlowCal/excel_ui.py

```diff
@@ -1126,23 +1181,24 @@ def generate_histograms_table(samples_table, samples, max_bins=1024):
             if pd.notnull(samples_table[header][sample_id]):
                 # Get units in which bins are being reported
                 unit = samples_table[header][sample_id]
-                # Decide whether to produce histograms in linear or log scale
-                if (unit != 'Channel') and \
-                    bool(sample.amplification_type(channel)[0]):
+                # Decide which scale to use
+                # Channel units result in linear scale. Otherwise, scale is
+                # decided based on amplification type.
+                if unit == 'Channel':
+                    scale = 'linear'
+                elif sample.amplification_type(channel)[0]:
                     scale = 'log'
                 else:
-                    scale = 'linear'
+                    scale = 'logicle'
```

Comment:
same as mentioned elsewhere: I think amplifier scale and plotting (and gating) scale are orthogonal, so I would still go with logicle gating, even with a log amplifier. if `unit` is "Channel", though, then I agree with what @castillohair has said elsewhere that user probably wants raw data, and I think linear scale is appropriate for that.

same as mentioned elsewhere: don't remember when `sample.amplification_type(channel)[0]` evaluates to `True`. comments and code imply a relationship, but would be nice if it was more explicit